### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aismann**
Correct wording/writting

### CODE REVIEW

File: external/README.md [hunk 1]
```diff
@@ -64,12 +64,12 @@
 - Version: 1.12.0
 - License: Apache-2.0
 
-## freetype
+## Freetype
```
Comment (halx99) [inline external/README.md:67]: shoulde be FreeType?

File: external/README.md [hunk 2]
```diff
@@ -64,12 +64,12 @@
...
 - Upstream: https://www.freetype.org/
 - Version: 2.10.4
 - License: BSD-style (The FreeType Project)
 
-## glad
+## GLAD
```
Comment (halx99) [inline external/README.md:72]: Glad?

File: external/README.md [hunk 3]
```diff
@@ -141,12 +141,12 @@
   - Version: git 1.3.7-83a82dd (1569)
   - License: BSD-3-Clause
 
-## openal
+## OpenAL
```
Comment (halx99) [inline external/README.md:144]: should be OpenAL Soft?

File: external/README.md [hunk 4]
```diff
@@ -226,7 +226,7 @@
 - Version: 1.2.11
 - License: zlib
 
-## iconv (libiconv)
+## libiconv (libiconv)
```
Comment (halx99) [inline external/README.md:229]: Does the (libiconv) could be removed?

**Review by halx99 — CHANGES_REQUESTED** [review]

File: external/README.md (hunk 1 above)
Comment (aismann) [reply external/README.md:67]: Please add the whole request. I will look later for correct wording.
Thanks.

File: external/README.md (hunk 4 above)
Comment (aismann) [reply external/README.md:229]: Please add the whole request. I will look later for correct wording.
Thanks.

**Comment by halx99** [comment]
others looks good to me

**Review by halx99 — APPROVED** [review]

**Pull request merged**